test/lib/bamboo/adapters/send_grid_adapter_test.exs

```diff
@@ -54,6 +64,11 @@ defmodule Bamboo.SendGridAdapterTest do
     end
   end
 
+  @doc """
+  sendgrid secret example dynamic
```

Comment:
What do you think about either expanding on this and explaining it more, or dropping it entirely if it's kind of clear what this is here for?

Reply:
The doc or the function? I will expand the doc

Reply:
Expanded in the latest changeset